bindings/fortran/CMakeLists.txt

```diff
@@ -16,6 +16,20 @@ target_link_libraries(dlite-fortran
   )
 
 
+## create list of modules to install from ${sources}
+set(modules "")
+foreach(source ${sources})
+  string(REPLACE ".f90" ".mod" module ${source})
+  if(EXISTS ${CMAKE_CURRENT_BINARY_DIR}/${module})
```

Comment:
This foreach look will run at the configuration stage. At that stage we would have no modules in the binary dir, and will therefore not add them to `modules`.

Can't we simply remove the if-statement?

Reply:
The foreach loop works, but I think we can remove the if-statement yes, it should be redundant. There was a bug in the first commit, will make an update

Reply:
Yes, the if-statement must be removed, otherwise the modules will not be installed (unless you reconfigure and install after you have first build the modules once).